### PR TITLE
Charts: update 'Customize Points and Labels' demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md [hunk 1]
```diff
@@ -1 +1,20 @@
-This demo shows how you can customize individual points and labels in&nbsp;the Chart.
\ No newline at end of file
+This demo shows how you can customize individual points and labels in the Chart component.
+
+## Customize Points
+
+Use the [customizePoint](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizePoint) function to change the appearance of individual series points. This function should return an object with properties that you want to change in a [point](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/) configuration. This demo uses this function to color all the points above the high average red, and the points below the low average blue.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md]: `This demo uses this function to color all the points above the high average red, and the points below the low average blue.`

I'd like to add a noun after "high average" that describes this dotted line. For example, is it technically correct to say "high average marker (red)" or "high average line (red)"?
Please clarify this for "low average" as well.

File: JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+
+## Customize Labels
+
+To customize the appearance of individual point labels, implement the [customizeLabel](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizeLabel) function. This function also should return an object with properties that need to be changed for a certain [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/label/). This demo specifies custom labels for the red points.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md]:
```suggestion
To customize the appearance of individual point labels, implement the [customizeLabel](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizeLabel) function. This function should also return an object with properties that need to be changed for a certain [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/label/). This demo specifies custom labels for the red points.
```

File: JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md (hunk 1 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md]: I rewrote this sentence a bit, could you please have a look?

File: JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md [hunk 3]
```diff
@@ -1 +1,20 @@
-This demo shows how you can customize individual points and labels in&nbsp;the Chart.
\ No newline at end of file
+This demo shows how you can customize individual points and labels in the Chart component.
+
+## Customize Points
+
+Use the [customizePoint](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizePoint) function to change the appearance of individual series points. This function should return an object with properties that you want to change in a [point](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/) configuration. This demo uses this function to color all the points with value above the high average number red, and the points with value below the low average number blue.
```
Comment (albertov05) [inline JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md]:
```suggestion
Use the [customizePoint](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#customizePoint) function to change the appearance of individual series points. This function should return an object with properties that you want to change in a [point](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/) configuration. This demo uses this function to color all points with values above the "High Average" in red, and points with values below the "Low Average" in blue.
```

File: JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md (hunk 1 above)
Comment (albertov05) [reply JSDemos/Demos/Charts/CustomizePointsAndLabels/description.md]: Thanks, Vlada. I added a suggestion to update your new version. Please take a look.

**Pull request merged**